### DISCUSSION TITLE
Google calendar not shown

### DISCUSSION THREAD

**Discussion by doctorcz** [MMRIZE/MMM-CalendarExt3#129] [Q&A] [answered]
Hello. first of all thanks for this great module. please advise, I can't fill the monthly calendar with my google calendar, as it is on the top left. I am attaching the config and screenshot. can someone help me please? thank you very much Mirek
[config.txt](https://github.com/MMRIZE/MMM-CalendarExt3/files/13840889/config.txt)
![mirror](https://github.com/MMRIZE/MMM-CalendarExt3/assets/33810585/e63367ce-8cd5-46f7-916a-6f5650ac5692)

**Comment by eouia** [comment, marked as answer]
Sorry for late reply.
Check `broadcastEvents: true`, the config value of `MMM-GoogleCalendar`.
